pytket/pytket/wasm/wasm.py

```diff
@@ -34,6 +36,110 @@ def __init__(self, filepath: str):
 
         self._wasmuid = hashlib.md5(self._wasm_file_encoded).hexdigest()
 
+        # check if the file is valid to run
+        if not Module.validate(Store(), base64.decodebytes(self._wasm_file_encoded)):
+            raise ValueError("wasm file not valid")
+
+        wasm_module = Module(Store(), base64.decodebytes(self._wasm_file_encoded))
```

Comment:
I thought it might be better to only store one of of the encodings and we need the base64 encoded variable later again in the submission. Happy to have both if you want that.

Reply:
Okay, great! That is what I have done now